src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -1000,6 +1033,106 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
+                        if (selectedConcept) {
+                          return selectedConcept.display;
+                        }
+                        return '';
+                      })()}
+                    />
+                    {(() => {
+                      if (!conceptToLookup) return null;
+                      if (isLoadingConcepts)
+                        return (
+                          <InlineLoading className={styles.loader} description={t('searching', 'Searching') + '...'} />
+                        );
+                      if (concepts?.length && !isLoadingConcepts) {
+                        return (
+                          <ul className={styles.conceptList}>
+                            {concepts?.map((concept, index) => (
+                              <li
+                                role="menuitem"
+                                className={styles.concept}
+                                key={index}
+                                onClick={() => handleOrdersSelect(concept)}
+                              >
+                                {concept.display}
+                              </li>
+                            ))}
+                          </ul>
+                        );
+                      }
+                      return (
+                        <Layer>
+                          <Tile className={styles.emptyResults}>
+                            <span>
+                              {t('noMatchingConcepts', 'No concepts were found that match')}{' '}
+                              <strong>"{debouncedConceptToLookup}".</strong>
+                            </span>
+                          </Tile>
+
+                          <div className={styles.oclLauncherBanner}>
+                            {
+                              <p className={styles.bodyShort01}>
+                                {t('conceptSearchHelpText', "Can't find a concept?")}
+                              </p>
+                            }
+                            <a
+                              className={styles.oclLink}
+                              target="_blank"
+                              rel="noopener noreferrer"
+                              href={'https://app.openconceptlab.org/'}
+                            >
+                              {t('searchInOCL', 'Search in OCL')}
+                              <ArrowUpRight size={16} />
+                            </a>
+                          </div>
+                        </Layer>
+                      );
+                    })()}
+                    {selectedOrders.length ? (
+                      <div>
+                        {selectedOrders.map((answer) => (
+                          <Tag className={styles.tag} key={answer.concept} type={'blue'}>
+                            {answer.label}
+                          </Tag>
+                        ))}
+                      </div>
+                    ) : null}
```

Comment:
Its better if we can have a way to remove selected ones, similar to how you've done when editing.